spec/services/articles/update_article_service_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Articles::Update do
+  subject(:article_update) { described_class.call(article.id, article_params) }
+
+  let(:article_params) { { title:, body:, status: Article.statuses.keys.second } }
+
+  let(:title) { 'title' }
+  let(:body) { 'some article body' }
+
+  let!(:article) { create(:article) }
+
+  describe '#call' do
+    context 'when success' do
+      context 'with updated comment' do
+        let(:article_attributes) do
+          article.reload.attributes.deep_symbolize_keys.except(:id, :created_at, :updated_at)
+        end
+
+        before { article_update }
+
+        it 'has valid data' do
+          expect(article_attributes).to eq(article_params)
+        end
+      end
+
+      it 'assigns comment' do
+        expect(article).to eq(article.reload)
+      end
+
+      it_behaves_like 'do not change model' do
+        let(:model) { Article }
+        let(:service_method) { article_update }
+      end
+    end
+
+    context 'when validation errors' do
```

Comment:
This context is inside `context 'when success'`

Context `context 'when success'` is redundant, `context 'with updated comment'` should be renamed to 'when success'